driver-core/src/main/com/mongodb/operation/MixedBulkWriteOperation.java

```diff
@@ -50,7 +50,6 @@
 import static com.mongodb.operation.OperationHelper.serverIsAtLeastVersionThreeDotSix;
 import static com.mongodb.operation.OperationHelper.validateWriteRequests;
 import static com.mongodb.operation.OperationHelper.withConnection;
-import static java.util.Collections.singletonList;
```

Comment:
Here's all the singleton lists that were being sent.

Reply:
Ack